test/e2e/kubernetes/kubernetes.go

```diff
@@ -251,3 +257,48 @@ func EnsureNodeWithIPIsDeleted(ctx context.Context, k8s *kubernetes.Clientset, i
 	}
 	return nil
 }
+
+func GetPodLogs(ctx context.Context, k8s *kubernetes.Clientset, name, namespace string) (string, error) {
+	req := k8s.CoreV1().Pods(namespace).GetLogs(name, &corev1.PodLogOptions{})
+	podLogs, err := req.Stream(ctx)
+	if err != nil {
+		return "", fmt.Errorf("opening log stream: %w", err)
+	}
+	defer podLogs.Close()
+
+	buf := new(bytes.Buffer)
+	if _, err = io.Copy(buf, podLogs); err != nil {
+		return "", fmt.Errorf("getting logs from stream: %w", err)
+	}
+
+	return buf.String(), nil
+}
+
+func ExecPod(ctx context.Context, config *restclient.Config, k8s *kubernetes.Clientset, name, namespace string, cmd []string) (string, string, error) {
```

Comment:
```suggestion
func ExecPod(ctx context.Context, config *restclient.Config, k8s *kubernetes.Clientset, name, namespace string, cmd ...string) (string, string, error) {
```
avoids creating a slice, so the caller looks a bit cleaner